### DISCUSSION TITLE
Update 0.7 Method name changes

### DISCUSSION THREAD

**Discussion by HeavensSword** [ramokz/phantom-camera#262] [Question / Help Wanted] [answered]
Hello I recently updated to the new 0.7 and discovered that a number of methods have had their names altered.

I have noticed that PhantomCamera2d has "append_follow_targets" and PhantomCamera3d has "append_follow_targets_node" and was curious if this is intended? They use to share the same name iirc.

I care mainly because I had created a C# wrapper for the plugin and have been having to update my mappings.

**Comment by ramokz** [comment, marked as answer]
Nope, that's an oversight. Releasing a hotfix shortly.

Thanks for calling this out :sparkles:
